layouts/404.html

```diff
@@ -0,0 +1,8 @@
+{{ define "main"}}
+<main id="main">
+    <div>
+        <h1 id="title"><a href="{{ "/" | relURL }}">Go Home</a></h1>
+        Sorry, This Page is not available.
```

Comment:
I think `This Page` should be without capitals

Reply:
Sorry! I've fixed it.